tests/vmi_multus_test.go

```diff
@@ -797,11 +797,18 @@ var _ = Describe("[Serial]SRIOV", func() {
 			startVmi(vmi)
 			waitVmi(vmi)
 
-			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			interfaceName, err := getInterfaceNameByMAC(vmi, mac)
-			Expect(err).NotTo(HaveOccurred())
+			vmiNamespace := vmi.Namespace
+			vmiName := vmi.Name
+			var interfaceName string
+			Eventually(func() error {
+				var err error
+				vmi, err = virtClient.VirtualMachineInstance(vmiNamespace).Get(vmiName, &metav1.GetOptions{})
+				if err != nil {
+					return err
+				}
+				interfaceName, err = getInterfaceNameByMAC(vmi, mac)
+				return err
+			}, 180, 1).Should(Succeed())
```

Comment:
can you use intervals set by time library?
```
}, 3*time.Minute, time.Second).Should(Succeed())
```

Reply:
Done, thanks